### PR TITLE
[flutter_markdown] Custom fontfeature superscript

### PR DESCRIPTION
**Pull request by dawidope**
Fix not solved issue from another PR: https://github.com/flutter/packages/pull/5058#issuecomment-1746670485.

I chose the 2nd option with custom font feature.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by dawidope** [comment]
@domesticmouse is there any blocker to merge it?

**Comment by domesticmouse** [comment]
PTAL @stuartmorgan

**Comment by stuartmorgan** [comment]
@dawidope Is this PR still something you are planning on updating to make it landable?

File: packages/flutter_markdown/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.6.18+4
+
+* Allow for choosing custom font feature to create superscript in footnotes when the font does not support `supr` font feature. Use `superscriptFontFeatureTag` property in `MarkdownStyleSheet`. For example for `Roboto` font you can set `numr`.
```
[the remark this answers is not in the document]
Comment (dawidope) [reply packages/flutter_markdown/CHANGELOG.md]: Fixed, let me know if it is ok now.

**Comment by dawidope** [comment]
@stuartmorgan I have done all requested changes. I hope that the changelog follows rules now.

**Review by tarrinneal — APPROVED** [review]
Seems good once you bump the version number and handle the conflicts.

**Comment by dawidope** [comment]
Done. Thanks!

**Pull request merged**